### PR TITLE
Update linting

### PR DESCRIPTION
**Pull request by pascalduez**
- Udpate ESlint dependency
- Add a base minimal config extending `eslint:recommended`
- Fix linting errors project wise

### CODE REVIEW

**Comment by ziir** [comment]
Yes! Thank you @pascalduez

File: app/index.js [hunk 1]
```diff
@@ -22,7 +24,7 @@ app.set('view engine', 'html');
 nunjucks.configure(path.resolve(__dirname, 'views'), {
   autoescape: true,
   watch: (app.get('env') === 'development'),
-  express: app
+  express: app,
```
Comment (esion) [inline app/index.js]: are you sure about this kind of rule ?

**Comment by esion** [comment]
I am not convinced about ending comma everywhere, it's even better for source diff, but is not fully supported by native javascript and totally not valid in json.

**Comment by pascalduez** [comment]
It's a stylistic rule, so yes we would need to discuss/validate it all together. 
It's totally fine in JS since ES3.  

Pros:
- Better git diff 
- Easier lines re-ordering
- Default style rule in several languages (Python, PHP, Sass, ...)
- Several "notorious" ESlint configs (Airbnb, ...) are enforcing it

Cons:
- Not well supported by old IE ( < 8 ) 


I would personally go for it, but will be ok with removing it.

Let's ping @ziir, @themouette, @KillianKemps

**Comment by ziir** [comment]
I think @pascalduez said it all :)

@esion I used to be very doubtful about this a few months ago, just like you.
I now realize it has made my life easier.
It's no big deal to have it or not in this project, but are you willing to give it a try ? :)

**Comment by esion** [comment]
:+1:  I give a try for everything :)

**Pull request merged**